tests/kafkatest/services/streams.py

```diff
@@ -413,13 +413,17 @@ def __init__(self, test_context, kafka):
                                                                  "org.apache.kafka.streams.tests.StreamsUpgradeTest",
                                                                  "")
         self.UPGRADE_FROM = None
+        self.UPGRADE_TO = None
 
     def set_version(self, kafka_streams_version):
         self.KAFKA_STREAMS_VERSION = kafka_streams_version
 
     def set_upgrade_from(self, upgrade_from):
         self.UPGRADE_FROM = upgrade_from
 
+    def set_upgrade_to(self, upgrade_to):
+        self.UPGRADE_TO = upgrade_to
```

Comment:
Where is `UPGRADE_TO` used?

Reply:
Forgot to revert this -- needed for version probing system test